### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by nitin-rachabathuni**
Please always provide the Jira Issue your PR is for, as well as test URLs where your change can be observed (before and after):

Jira ID: https://jira.corp.adobe.com/browse/EXLM-6

Test URLs:

- Before: https://main--exlm--adobe-experience-league.hlx.page/docs/integrations-learn/experience-cloud/solution-categories/content-management
- After: https://feature-exlm-250-footer-links--exlm--adobe-experience-league.hlx.page/docs/authoring-guide-exl/using/markdown/cheatsheet

### CODE REVIEW

**Review by tprasad05 — APPROVED** [review]

File: blocks/footer/footer.js [hunk 1]
```diff
@@ -1,28 +1,146 @@
...
+    if (a.title?.toLowerCase() === 'home') {
+      a.innerHTML = `<span class="icon icon-home"></span>`;
+    }
+  });
+}
+
+function decorateCopyrightsMenu() {
+  const footerLastRow = document.createElement('div');
+  footerLastRow.classList.add('footer-last-row');
+  const footerLangSocial = document.querySelector('.footer-lang-social');
```
Comment (noormd08) [inline blocks/footer/footer.js:95]: Dont use document.querySelector, instead pass the footer element and get it from there.

File: blocks/footer/footer.js [hunk 2]
```diff
@@ -1,28 +1,146 @@
...
     // decorate footer DOM
     const footer = document.createElement('div');
     footer.innerHTML = html;
-
-    decorateIcons(footer);
+    decorateMenu(footer);
+    decorateSocial(footer);
+    decorateBreadcrumb(footer);
     block.append(footer);
+    decorateCopyrightsMenu();
```
Comment (noormd08) [inline blocks/footer/footer.js:143]: Pass footer element as param and make use of it inside the function for querying the child elements.

File: blocks/footer/footer.js (hunk 1 above)
Comment (thomsebastin) [reply blocks/footer/footer.js:95]: @nitin-rachabathuni Please consider this comment in your refactor PRs.

**Comment by noormd08** [comment]
@nitin-rachabathuni The PR is merged. Please take care of review comments in the next PR

**Comment by nitin-rachabathuni** [comment]
> @nitin-rachabathuni The PR is merged. Please take care of review comments in the next PR

Sure @noormd08 and @thomsebastin

**Pull request merged**